public/src/components/Members.tsx

```diff
@@ -90,7 +90,7 @@ const Members = (): JSX.Element => {
                     <Search />
                 </div>
                 <div className="ml-auto">
-                    <div className={edit ? "btn btn-sm btn-danger" : "btn btn-sm btn-primary"} onClick={() => setEditing(!edit)}>
+                    <div className={edit ? "btn btn-danger" : "btn btn-primary"} onClick={() => setEditing(!edit)}>
                         {edit ? "Cancel" : "Edit"}
```

Comment:
I think the `Cancel` button should be renamed `Done` because if you've accepted or rejected a student, there is no cancelation option.